src/azure/Model/MethodPya.cs

```diff
@@ -51,14 +51,22 @@ public bool IsLongRunningOperation
             get { return Extensions.ContainsKey(AzureExtensions.LongRunningExtension); }
         }
 
+        public bool HasCloudError => (CodeModel as CodeModelPya).AzureArm && (DefaultResponse.Body == null || DefaultResponse.Body.Name == "CloudError");
+
+        public bool HasHttpResponseError => !(CodeModel as CodeModelPya).AzureArm && DefaultResponse.Body == null;
```

Comment:
`as` will return null if the cast fails. If you are not going to check for null:ness, I'd just to a straight-up cast...

Reply:
Changed to cast